src/components/header/headerMenu/HeaderMenu.jsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import { Button, ButtonGroup } from '@mui/material';
 import { getFieldLabel } from '../../../utils';
 
 function HeaderMenu() {
   return (
     <ButtonGroup variant="text" aria-label="text button group" color="primary">
-      <Button>{getFieldLabel('header.menu.button.internships')}</Button>
-      <Button>{getFieldLabel('header.menu.button.candidates')}</Button>
-      <Button>{getFieldLabel('header.menu.button.letters')}</Button>
+      <Button>
+        <Link to="/internships">
```

Comment:
Please read about buttons and links. https://css-tricks.com/a-complete-guide-to-links-and-buttons/

Reply:
Also pay attention https://mui.com/api/button/
![image](https://user-images.githubusercontent.com/75979545/140476261-bc58de14-7c6b-4a29-ae16-d5cdc39ad8d4.png)